clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -98,6 +98,19 @@
                                                          + "elapses the client will resend the request if necessary or fail the request if "
                                                          + "retries are exhausted.";
 
+    public static final String DEFAULT_LIST_KEY_SERDE_INNER_CLASS = "default.list.key.serde.inner";
```

Comment:
Should I add a `DOC` for `DEFAULT_LIST_KEY/VALUE_SERDE_INNER_CLASS` configs? 

I was looking at `DEFAULT_WINDOWED_KEY/VALUE_SERDE_INNER_CLASS` configs in `StreamsConfig` class, and they don't have docs underneath them.

Reply:
Sound like something we should fix, ie, add corresponding DOC entries to `StreamsConfig`